commands/publish.ts

```diff
@@ -67,7 +68,7 @@ export default class PublishCommand extends Command {
       await spinner.fail('Publish failed');
       throw error;
     }
-    let pkg = require(path.join(process.cwd(), 'package.json'));
+    let pkg = readPkg();
```

Comment:
looks like you are using the async version here without `await` https://www.npmjs.com/package/read-pkg#readpkgpath-options